Gemfile

```diff
@@ -17,6 +17,7 @@ gem "jquery-rails"
 # gem 'debugger'
 
 group :test do
+  gem 'test-unit'
```

Comment:
Ruby 2.x doesn't include test unit anymore.  Or something similar?  http://stackoverflow.com/questions/28252036/how-to-run-existing-test-code-on-ruby-2-2